tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -710,6 +722,11 @@ public bool DiagnosticSourceEnabled
         /// </remarks>
         internal int OutgoingTagPropagationHeaderMaxLength { get; }
 
+        /// <summary>
+        /// Gets a value indicating the injection and extraction propagation styles.
+        /// </summary>
+        internal string[] PropagationStyle { get; }
```

Comment:
I don't think we need this one either. We're just changing the default, not adding new settings. 😄